.eslint/rules/no-invalid-paste-module-paths.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'Forbid invalid paste package module paths',
+      category: 'Possible Problems',
+    },
+    fixable: null,
+    schema: [], // no options
+    messages: {
+      src: 'Invalid module path using "src" path. Use a bare module specifier instead.',
+    },
+  },
+
+  create(context) {
+    const check = ({value}, node) => {
+      if (value.startsWith('@twilio-paste') && value.includes('src')) {
+        context.report({messageId: 'src', node});
+      }
+    };
+
+    return visitor(check);
+  },
+};
+
+const visitor = (func) => {
+  const checkSourceValue = (source, node) => {
+    if (source == null) return;
+    func(source, node);
+  };
+
+  const checkSource = (node) => {
+    checkSourceValue(node.source, node);
+  };
+
+  const checkImportCall = (node) => {
+    let modulePath;
+
+    if (node.type === 'ImportExpression') {
+      modulePath = node.source;
+    }
+
+    if (node.type === 'CallExpression' && node.callee.name === 'require' && node.arguments.length === 1) {
+      modulePath = node.arguments[0];
+    }
+
+    if (modulePath == null) return;
+    if (modulePath.type !== 'Literal') return;
+    if (typeof modulePath.value !== 'string') return;
+
+    checkSourceValue(modulePath, node);
+  };
+
+  return {
```

Comment:
Suggestion: this all looks good to me, but would you mind adding a couple comments explaining how this works? I think I get it, but I was just googling around haha